src/pages/resultsView/pathwayMapper/PathWayMapperContainer.tsx

```diff
@@ -9,14 +9,66 @@ import { ResultsViewPageStore } from 'pages/resultsView/ResultsViewPageStore';
 import ResultsViewURLWrapper from 'pages/resultsView/ResultsViewURLWrapper';
 import { AppStore } from 'AppStore';
 import { useLocalObservable } from 'mobx-react-lite';
-import { runInAction } from 'mobx';
+import { runInAction, autorun } from 'mobx';
+import request from 'superagent';
+import { remoteData } from 'cbioportal-frontend-commons';
 
 interface IPathwayMapperContainerProps {
     resultsViewPageStore: ResultsViewPageStore;
     appStore: AppStore;
     urlWrapper: ResultsViewURLWrapper;
 }
 
+interface ResultItem {
+    databases: DatabaseItem[];
+}
+interface DatabaseItem {
+    name: string;
+    numberOfNetworks: string;
+    url: string;
+}
+interface ApiResponse {
+    results: ResultItem[];
+}
+
+function makeRemoteData() {
+    return remoteData<DatabaseItem[]>({
+        await: () => [],
+        invoke: async () => {
+            try {
```

Comment:
@SAHU-01 thanks very much for making this change.  a couple more things. it would be best not to catch the error here.  what will then happen is the the promise will will have isError=true.  You can then use that in the tooltip UI to show an error message to the user, inside the tooltip, which will explain why there are no results.